File: src/java/WebNinja/src/test/java/webninja/selenuim/ReportsPage.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ReportsPage {
    private final WebDriver webDriver;

    public ReportsPage(WebDriver webDriver) {
        this.webDriver = webDriver;
    }

    public IssueTable showIssuesAssignedToLoggedInUser() {
        webDriver.findElement(By.xpath("//div[@id='bodyFram']/div[3]/ul/li[2]/a")).click();
        return new IssueTable(webDriver);

    }
}

File: src/dotnet/WebNinja/WebNinja/selenium/ReportsPage.cs
using OpenQA.Selenium;

namespace WebNinja.selenium
{
    public class ReportsPage : PageObject
    {
        public ReportsPage(IWebDriver driver) : base(driver)
        {
        }

        public IssueTable ShowIssuesAssignedToLoggedInUser()
        {

            var element = Driver.FindElement(By.XPath("//div[@id='bodyFrame']/div[3]/ul/li[2]/a"));
            element.Click();
            return new IssueTable(Driver);
        }
    }
}
